packages/office-ui-fabric-react/src/components/Slider/Slider.test.tsx

```diff
@@ -16,8 +18,8 @@ describe('Slider', () => {
   });
 
   it('renders a slider', () => {
-    const component = ReactTestUtils.renderIntoDocument(
-      <Slider label='slider' />
+    const component = ReactTestUtils.renderIntoDocument<SliderBase>(
```

Comment:
**Note:** that I've refactored the tests to test `<SliderBase/>` as we did in https://github.com/OfficeDev/office-ui-fabric-react/commit/3626565be858e54cbb8ddcb617d62fa52823254a

Reply:
I really don't completely understand this. What is the reasoning?

Reply:
I was trying to follow a component's tests that followed the `getStyles` pattern and were passing. Seems inconsistent tho and perhaps I followed the wrong pattern. Prior to using `<SliderBase/>` the DOM wouldn't render, likely because I was omitting something that was causing `<Slider/>` to throw.

Reply:
Re: the type parameter changes those can be removed and won't affect anything, just happened to be the pattern that I followed from https://github.com/OfficeDev/office-ui-fabric-react/commit/3626565be858e54cbb8ddcb617d62fa52823254a but even that is inconsistent.